### PR TITLE
Finish up the new website.

### PR DESCRIPTION
**Pull request by thruflo**
Flesh out and migrate content from the previous Electric website to the "next" site.

### CODE REVIEW

File: docs/.vitepress/config.mts [hunk 1]
```diff
@@ -81,7 +81,7 @@ export default defineConfig({
     siteTitle: false,
     socialLinks: [
       { icon: 'discord', link: 'https://discord.electric-sql.com' },
-      { icon: 'github', link: 'https://github.com/electric-sql/electric' }
+      { icon: 'github', link: 'https://github.com/electric-sql' }
```
Comment (KyleAMathews) [inline docs/.vitepress/config.mts:84]: this is just to the org — is that what you wanted?

File: docs/.vitepress/theme/home-features-before.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Sync your Postgres data <span class="no-wrap">wherever you need it</span>
+
+Electric syncs [little subsets](/guides/shapes) of your Postgres data into local apps<span class="no-wrap-sm"><span class="hidden-xs">, services</span> and environments</span>. <span class="hidden-sm"><span class="no-wrap-lg">So you can have the data you need</span>, <span class="no-wrap">in-sync</span>, <span class="no-wrap">wherever you need it</span>.</span>
```
Comment (KyleAMathews) [inline docs/.vitepress/theme/home-features-before.md:3]: love "little subsets"!

File: docs/.vitepress/theme/home-cta.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <a href="/api/integrations/react">
+      framework&nbsp;hooks</a></span>.
+</span>
+
+```tsx
+import { useShape } from '@electric-sql/react'
+
+const Component = () => {
+  const { data } = useShape({
+    url: `/v1/shape/items`
```
[the remark this answers is not in the document]
Comment (KyleAMathews) [reply docs/.vitepress/theme/home-cta.md:27]: This needs to be a full URL

File: docs/.vitepress/config.mts (hunk 1 above)
Comment (thruflo) [reply docs/.vitepress/config.mts:98]: Yup, I have the general GitHub links to the org, which has electric and pglite pinned. Then the product tiles link to the specific repos with star count.

**Comment by thruflo** [comment]
@KyleAMathews I'm curious whether you're amenable to (or whether you want to cut) the two stage CTA section at the bottom of https://deploy-preview-1570--electric-next.netlify.app

It's intended to get across that Electric is both simple to get started with "one fetch at a time" and something that intends to take you all the way to a fully local-first paradigm when you're ready.

I'm conscious that I'm under strict instructions not to overreach / document things that aren't true yet. In my defence, the code shown does work and I think showing it may fire the imagination and connect the docs for developers, especially those who have come into electric through pglite.

**Comment by KyleAMathews** [comment]
Yeah the pglite code snippet seems like a good idea. It's a genuinely remarkable thing so both impressive and will get lots of people talking.

**Review by KyleAMathews — COMMENTED** [review]
🥳 :shipit:

**Review by KyleAMathews — APPROVED** [review]

**Pull request merged**